geeksurvey/templates/home.html

```diff
@@ -4,149 +4,147 @@
 <html>
   <head>
     {% include 'include_bootstrap.html' %}
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
-    <meta name="description" content="" />
-    <meta name="author" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"/>
+    <meta name="description" content=""/>
+    <meta name="author" content=""/>
```

Comment:
didnt see this before but we should probably write something for description and author. in this and base.html

Reply:
Given that multiple people have worked on the files, how should I fill the author? Also I'll working on the descriptions

Reply:
Actually given that different pages contain different content and have different authors, wouldn't it be best to edit the author and description meta tags in each individual page and not in the base.html file?

Reply:
hmm yea good point. We can either be cryptic and put the author as "SuperGeeks" for everything or just nuke the author tag.
I think the description tag should stay and get a nice sentence for SEO (it shows up on Google search results and helps get the site indexed)

Reply:
I'll go ahead and use "SuperGeeks" as the author as the project may be extended upon in the future by someone other than us. I'll go ahead and add a customized description for pages likely to show up in search engine results (home page, log in/sign up pages, etc.).